app/ide-desktop/lib/dashboard/src/authentication/src/config.tsx

```diff
@@ -8,7 +8,7 @@ import * as auth from "./authentication/config";
 // =================
 
 /** The current environment that we're running in. */
-export const ENVIRONMENT: Environment = "production";
+export const ENVIRONMENT: Environment = "pbuchu";
```

Comment:
Such things should never be in a PR that is ready for review. Please be sure that no temporary things are hardcoded in the code.